examples/zero_shot_imagenet.py

```diff
@@ -56,7 +56,7 @@ def main(rank: int = 0, world_size: int = 1):
     parser.add_argument('--split', default='validation',
                         help='The dataset split to use.'
     )
-    parser.add_argument('--resize-multiple', type=int, default=None,
+    parser.add_argument('--resize-multiple', type=int, default=16,
```

Comment:
I think `visualize_features` also needs this

Reply:
`knn_classification` and `position_predictor` too, I think

Reply:
Good catch. Done.